### ISSUE TITLE
gj/gk not working in wrapped lines

### ISSUE THREAD

**Issue by DrakeXiang** [asvetliakov/vscode-neovim#90]
As the title says `gj` and `gk` now behaves the same as normal `j` and `k`

**Comment by asvetliakov** [comment]
it's not possible to determine width of vscode viewport so i'm setting neovim width to large value (500) and forcing ```set nowrap``` option. This probably won't be fixed for long time, sorry

Vote for [this vscode request](https://github.com/microsoft/vscode/issues/58954)

**Comment by asvetliakov** [comment]
Workaround:
```
nnoremap gk :<C-u>call rpcnotify(g:vscode_channel, 'vscode-command', 'cursorMove', { 'to': 'up', 'by': 'wrappedLine' })<CR>
nnoremap gj :<C-u>call rpcnotify(g:vscode_channel, 'vscode-command', 'cursorMove', { 'to': 'down', 'by': 'wrappedLine' })<CR>
```

**Comment by asvetliakov** [comment]
Will add to next release

**Comment by DrakeXiang** [comment]
Big thanks for the workaround and all the effort! I wish I could replace my old vim extension with this and give more feedback, the only thing that stops me so far is the Chinese character issue 😏

**Comment by jweickm** [comment]
Hi, 
I saw that this was fixed in [dbfb5ef](https://github.com/vscode-neovim/vscode-neovim/commit/dbfb5ef5bc819c0d0811d3ae3dbfce85eac06412) for the normal mode. However, for the visual mode, the same issue still persists. Moving with `gj` and `gk` results in moving the entire line (not just the visual line). 
Is there any fix for this? It would be amazing if this could be looked at again.
